neurogym/config/config.py

```diff
@@ -94,6 +98,21 @@ def settings_customise_sources(
     @field_validator("*", mode="before", check_fields=False)
     @classmethod
     def expand_env_vars(cls, value: str) -> str:
+        """Expand environment variables in fields before validation.
+
+        Applies to all fields and supports strings, lists of strings, and
+        dictionaries with string values. Uses `os.path.expandvars` to replace
+        placeholders like "$HOME" or "${USER}" with their actual values.
+
+        This allows configuration files to contain dynamic paths such as:
+            local_dir = "${HOME}/neurogym_outputs"
```

Comment:
I would make that a subdirectory of `.cache` (`${HOME}/.cache/neurogym_outputs`). I would personally find it very annoying if a package adds a non-hidden directory at the top level of my HOME. 🙂